### PR TITLE
Add management api functionality

### PR DESCRIPTION
**Pull request by sachithKay**

### CODE REVIEW

File: components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            service = transitGhostServicesMap.get(serviceName);
+        }
+
+        if (service == null) {
+            String msg = "Invalid service name, service not found : " + serviceName;
+            throw new AxisFault(msg);
+        }
+
+        // if the existing service is a ghost service, deploy the actual one
+        if (GhostDeployerUtils.isGhostService(service)) {
```
Comment (Nirothipan) [inline components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java]: We have removed this . We can skip this .

File: components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+package org.wso2.micro.service.mgt;
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.description.AxisOperation;
+import org.apache.axis2.description.AxisService;
+import org.apache.axis2.description.AxisServiceGroup;
+import org.apache.axis2.description.Parameter;
+import org.apache.axis2.engine.AxisConfiguration;
+import org.wso2.carbon.service.mgt.util.Utils;
```
Comment (Nirothipan) [inline components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java]: We don't have this class AFAIK

File: components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java (hunk 1 above)
Comment (sachithKay) [reply components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java]: +1. done

File: components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java (hunk 2 above)
Comment (sachithKay) [reply components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/service/mgt/ServiceAdmin.java]: import is corrected

**Pull request merged**